addons/dark-www/experimental_scratchr2.css

```diff
@@ -384,9 +382,9 @@ ul.pagination span /* mobile forums */ {
   color: var(--darkWww-button-scratchr2ButtonText);
 }
 .button {
-  background-image: linear-gradient(var(--darkWww-button), var(--darkWww-button-variant));
-  border-color: rgba(0, 0, 0, 0.2);
-  border-bottom-color: rgba(0, 0, 0, 0.3);
+  background-color: var(--darkWww-button-scratchr2);
+  background-image: none; /* Fixes Scratch bug: https://scratch.mit.edu/discuss/topic/694218/ */
```

Comment:
Wow, hasn't this been fixed yet?

Reply:
Doesn't seem to be.
![image](https://github.com/ScratchAddons/ScratchAddons/assets/68464103/dcccddf4-5144-4616-891a-c2fe24c1a3be)

Reply:
This bug was never visible with `dark-www` enabled. I had to add `background-image: none` to fix it in this PR because I removed the gradient from purple buttons.